tests/testthat/test-shared-secret.R

```diff
@@ -11,7 +11,7 @@ test_that("requests with shared secrets pass, w/o fail", {
   res <- PlumberResponse$new()
   output <- pr$route(req, res)
   expect_equal(res$status, 400)
-  expect_equal(output, list(error = "400 - Bad request"))
+  expect_equal(output, list(error = "400 - Bad request", message = "Shared secret mismatch"))
```

Comment:
This should be the original code. I'd rather have it error and fixed elsewhere

```suggestion
  expect_equal(output, list(error = "400 - Bad request")
```